DOC-2194: fix documentation for TINY-10249 in the 6.7.2 release notes. (#2948)
tinymce/tinymce-docs: modules/ROOT/pages/6.7.2-release-notes.adoc

// CCFR

=== Removed use of async for editor rendering which caused visual blinking when reloading the editor in-place.
// TINY-10249

> In contrast to {productname} version 5, where the editor would initiate rendering without waiting for the completion of CSS loading, in version 6, the editor instance now deliberately synchronizes with the loading of CSS. This modification aims to prevent flickering by ensuring that the necessary styles are fully loaded before proceeding with the rendering process.
> 
> As a result the browser was able to render a partially complete editor, resulting in UI flicker if an editor was reloaded.
> 
> {productname} 6.7.2 addresses this, by once again blocking the render function from loading the CSS in parallel with other actions necessary for displaying the editor.
> 
> As a result, the editor no longer renders the flicker previously displayed when the editor is refreshed.

Author: ```suggestion
In contrast to {productname} version 5, where the editor would initiate rendering without waiting for the completion of CSS loading, in version 6, the editor instance now deliberately synchronizes with the loading of CSS. This modification aims to prevent flickering by ensuring that the necessary styles have finished loading before proceeding with the rendering process.

{productname} 6.7.2 addresses this, by once again blocking the render function from loading the CSS in parallel with other actions necessary for displaying the editor.

As a result, the editor no longer renders the flicker previously displayed when the editor is refreshed.
```

> === css loading in `toolbardrawertoggletest` flickering.

Author: @TheSpyder Suggested title update to reflect changes.

```suggestion
=== Partial rendering of editor css caused UI flicker during editor loading
```

// TINY-10249

> Previously, when a {productname} editor instance would load the content CSS for the `toolbardrawertoggletest` It would cause a noticeable flicker to the editor's UI.

Reviewer: Actually it's not the whole thing that's wrong, just this line.

It's unrelated to the test - that's the part I'm complaining about. It's just when the editor is reloaded.

It's a simple regression; TinyMCE 5 didn't flicker when the editor reloaded and TinyMCE 6 does.

Author: Updated with https://github.com/tinymce/tinymce-docs/pull/2948#discussion_r1367986706 copy edit suggestion.

> Previously, when a {productname} editor instance would load the content CSS for the `toolbardrawertoggletest` It would cause a noticeable flicker to the editor's UI.
> 
> As a result, the CSS would take a few milliseconds to load, resulting in the UI flicker during a editor refresh.

Reviewer: I'm not sure how we document regressions, try this?
```suggestion
Unlike {productname} version 5, when a {productname} version 6 editor instance was loaded it asynchronously waited for the editor styles and content to load.

As a result the browser was able to render a partially complete editor, resulting in UI flicker if an editor was reloaded.
```

> {productname} 6.7.2 addresses this, by adding a fix that now allows the render function to load the CSS in parallel with other necessary actions for displaying the editor.

Reviewer: Also this is backwards

```suggestion
{productname} 6.7.2 addresses this, by once again blocking the render function from loading the CSS in parallel with other actions necessary for displaying the editor.
```
